test/package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "scripts": {
     "test": "node --preserve-symlinks test.js test",
-    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm test",
-    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcov npm test",
+    "coverage": "cross-env NODE_ENV=test ESM_OPTIONS='{cache:\"node_modules/.cache/esm-coverage\"}' nyc --reporter=lcov --reporter=text-summary npm test",
```

Comment:
nice, can this be part of the package.json or other .npmFOO thing?

Reply:
I would like to make this more DRY, but I wasn't sure how that didn't add complexity.  We only want ESM_OPTIONS set for coverage commands, so any sort of global config doesn't seem like it would work.  I had tried a cross-env-shell version that had a single common command and passed args to it, but that seemed more complex.  I pushed a patch to change to disable the cache. Perhaps in the future esm can be fixed or ESM_DISABLE_CACHE=true will work for this use case.